guiEditor/src/main.scss

```diff
@@ -58,7 +58,7 @@
     display: grid;
     grid-template-rows: 1fr 40px auto 40px;
     grid-template-columns: 100%;
-    height: 100%;
+    height: calc(100% - 55px);

```

Comment:
the 55 should be stored as a variable as well, incase you decide to change the header's height